docs/howto/deployment/wsgi/modwsgi.txt

```diff
@@ -97,44 +97,96 @@ should put in this file, and what else you can add to it.
+location of the Apache APXS script for your Apache installation before
+performing the installation. If you are using ``Linux``, ``macOS`` or other
+``UNIX`` type operating system, and you don't have Apache available, you can
+instead install ``mod_wsgi`` using:
 
-.. _details on setting up daemon mode: https://modwsgi.readthedocs.io/en/develop/user-guides/quick-configuration-guide.html#delegation-to-daemon-process
+.. code-block:: apache
+
+    pip install mod_wsgi-standalone
+
+When installing mod_wsgi-standalone, it will also install a version of Apache
+into your Python distribution. You can only use mod_wsgi-express when using
+this variant of the package. The mod_wsgi-standalone package follows the same
+version numbering as the mod_wsgi package on PyPi. If you are on Windows and
+your Apache distribution is not installed into the directory C:/Apache24, first
+set the environment variable MOD_WSGI_APACHE_ROOTDIR to the directory
+containing the Apache distribution. Ensure you use forward slashes in the
+directory path. The directory path should not include path components with
+spaces in the name. Note that nothing will be copied into your Apache
+installation at this point. As a result, you do not need to run this as the
+root user unless installing it into a site wide Python installation rather than
+a Python virtual environment. It is recommended you always use Python virtual
+environments and never install any Python package directly into the system
+Python installation. On a UNIX type system, to verify that the installation was
+successful, run the mod_wsgi-express script with the start-server command:
+
+.. code-block:: apache
+
+    mod_wsgi-express start-server
+
+This will start up Apache/mod_wsgi on port 8000. You can then verify that the
+installation worked by pointing your browser at:
+
+.. code-block:: apache
+
+    http://localhost:8000/
+
+When started in this way, the Apache web server will stay in the foreground. To
+stop the Apache server, use CTRL-C. For a simple WSGI application contained in
+a WSGI script file called wsgi.py, in the current directory, you can now run:
+
+.. code-block:: apache
+
+    mod_wsgi-express start-server wsgi.py
+
+This instance of the Apache web server will be completely independent of, and
+will not interfere with any existing instance of Apache you may have running on
+port 80. If you already have another web server running on port 8000, you can
+override the port to be used using the --port option:
+
+.. code-block:: apache
+
+    mod_wsgi-express start-server wsgi.py --port 8080
+
+For a complete list of options you can run:
+
+.. code-block:: apache
+
+    mod_wsgi-express start-server --help
+
+For further information related to using mod_wsgi-express see the main mod_wsgi documentation.
 
 .. _serving-files:
 
 Serving files
 =============
 
-Django doesn't serve files itself; it leaves that job to whichever web
+Django doesn't serve files itself; it leaves that job to whichever Web
```

Comment:
`Web` should not be capitalized, see [docs](https://docs.djangoproject.com/en/4.0/internals/contributing/writing-documentation/#commonly-used-terms).

Reply:
Not only in this line. Please revert changes to other lines where `web` is switched to `Web`.

Reply:
Aah, OK @felixxm Fixed with this commit https://github.com/django/django/pull/14939/commits/8034353caad11f06364b1a08181b7bd9d9b4a9ca